src/desktop/components/split_test/running_tests.coffee

```diff
@@ -21,10 +21,10 @@ module.exports = {
       control: 50
       experiment: 50
     edge: 'experiment'
-#  onboarding_test:
-#    key: 'onboarding_test'
-#    outcomes:
-#      control: 50
-#      experiment: 50
-#    edge: 'experiment'
+ onboarding_test:
+   key: 'onboarding_test'
+   outcomes:
+     control: 100
+     experiment: 0
+   edge: 'experiment'
```

Comment:
In order for `splitTest('onboarding_test').view()` to work, we need a test to track. I can add a conditional to `splitTest('onboarding_test').view()` only fire if there is a test, but that makes it impossible to QA and I figured since we're launching this tomorrow it doesn't matter.

Reply:
So this actually launches the test? If so I think I ought to hold off on merging this because I stilled to wrap up/fix the next button and @yuki24 and I noticed that something was up with the react router.

Reply:
@xtina-starr no, I don't think this launches it. It just means all users get old onboarding. Then the query param `personalize?onboarding_test=experiment` still works to force a version.

Reply:
Ahh I see now. Then perhaps we'd might also want to remove setting `sd.ONBOARDING_TEST` via a cookie [here](https://github.com/artsy/force/blob/master/src/lib/middleware/locals.coffee#L54-L57) in this PR because it seems redundant if we are using the splitTest component.

Reply:
Truee, and our internal team can still access via `?split_test[onboarding_test]=control`. Ok removing now!